Source/santactl/Commands/sync/SNTCommandSyncPreflight.m

```diff
@@ -105,12 +105,17 @@ - (BOOL)sync {
   self.syncState.FCMToken = resp[kFCMToken];
 
   // Don't let these go too low
-  NSUInteger value = [resp[kFCMFullSyncInterval] unsignedIntegerValue];
+  NSUInteger FCMintervalValue = [resp[kFCMFullSyncInterval] unsignedIntegerValue];
```

Comment:
`FCMIntegerValue`

Though as this variable is being re-used for multiple distinct purposes, value is still fine.

Reply:
I was going to leave it as value but I thought you might want them split - happy to do whatever you'd prefer 😄